Mathlib/Data/List/Rotate.lean

```diff
@@ -477,6 +477,24 @@ theorem IsRotated.map {β : Type*} {l₁ l₂ : List α} (h : l₁ ~r l₂) (f :
   rw [map_rotate]
   use n
 
+theorem IsRotated.cons_getLast_dropLast
+    (L : List α) (hL : L ≠ []) : (L.getLast hL :: L.dropLast) ~r L := by
```

Comment:
```suggestion
    (L : List α) (hL : L ≠ []) : L.getLast hL :: L.dropLast ~r L := by
```
If it doesn't work, please tell me and we will fix the notation precedences

Reply:
I'm sorry, but it really fails.

Reply:
Should be fixed after #15876

Reply:
Can you please merge master now?

Reply:
Merged. Sorry for reply lately...